### PR TITLE
Add Sync-My-L2P-app v2.2.0

### PR DESCRIPTION
**Pull request by mainrs**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download sync-my-l2p` is error-free.
- [x] `brew cask style --fix sync-my-l2p` left no offenses.
- [x] `brew cask install sync-my-l2p` worked successfully.
- [x] `brew cask uninstall sync-my-l2p` worked successfully.

### CODE REVIEW

File: Casks/sync-my-l2p.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'sync-my-l2p' do
+  version '2.2.0'
+  sha256 '76344b27cda005a3e60135d880f482eef08bc0f1db84480e39677dd230823886'
+
+  url 'https://github.com/Sync-my-L2P/Sync-my-L2P/releases/download/v2.2.0/SyncMyL2P-2.2.0-osx.dmg'
```
Comment (adidalal) [inline Casks/sync-my-l2p.rb]: Use #{version} and "" instead of hard-coding a URL.

Comment (adidalal) [inline Casks/sync-my-l2p.rb]: Also, mind adding an [appcast](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md)?

Comment (SirWindfield) [inline Casks/sync-my-l2p.rb]: Doing it right now...

Comment (adidalal) [inline Casks/sync-my-l2p.rb]: URL verified - you can add the comment (or just let me know and I'll do it before I merge)

`# github.com/Sync-my-L2P/Sync-my-L2P was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/4ad02c9f087deb9e553ab0129c079d73c474add8/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

Thanks!

Comment (SirWindfield) [inline Casks/sync-my-l2p.rb]: Could you help me out a bit? Hoe do I update my pull request? Just push the changes to my repo and re-run the travis ci ?

**Comment by SirWindfield** [comment]
One question: why exactly did the travic-ci build fail? I couldn't figure it out

**Comment by adidalal** [comment]
@SirWindfield We know :( https://github.com/caskroom/homebrew-cask/issues/21041

**Pull request merged**